pkg/build/builder/docker_test.go

```diff
@@ -258,6 +258,10 @@ func TestDockerfilePath(t *testing.T) {
 			},
 		}
 
+		getContainerNetworkConfigFunc := func() (string, string, error) {
```

Comment:
could you add a comment saying that it is okay to return an empty network configuration since the tests use a FakeDocker implementation that doesn't build anything?